testpar/t_subfiling_vfd.c

```diff
@@ -1513,6 +1519,13 @@ test_subfiling_write_many_read_one(void)
     fspace_id = H5Screate_simple(1, dset_dims, NULL);
     VRFY((fspace_id >= 0), "H5Screate_simple succeeded");
 
+#ifdef H5_HAVE_FILTER_DEFLATE
+    plist_id = H5Pcreate(H5P_DATASET_CREATE);
+    VRFY((plist_id >= 0), "H5Pcreate() succeeded");
+    chunk_dims[0] = dset_dims[0] / 2;
+    VRFY((H5Pset_chunk(plist_id, 1, chunk_dims) >= 0), "H5Pset_chunk succeeded");
+    VRFY((H5Pset_deflate(plist_id, 9) >= 0), "H5Pset_deflate succeeded");
+#endif
     dset_id = H5Dcreate2(file_id, "DSET", SUBF_HDF5_TYPE, fspace_id, H5P_DEFAULT, H5P_DEFAULT, H5P_DEFAULT);
```

Comment:
Should be,

dset_id = H5Dcreate2(file_id, "DSET", SUBF_HDF5_TYPE, fspace_id, H5P_DEFAULT, plist_id, H5P_DEFAULT);